js/insult.js

```diff
@@ -43,6 +49,13 @@
 		'Scrubber',
 		'Pusher',
 		'Scratcher',
+		'Shitter',
+		'Blister',
+		'Dragon',
+		'Fister',
+		'Skank',
```

Comment:
Love it, except we're only using double-syllable nouns. Remove line 56 and we're golden to merge this :)